bin/pycbc_live

```diff
@@ -337,7 +337,7 @@ parser.add_argument('--trim-padding', type=float, default=0.25,
 parser.add_argument("--enable-bank-start-frequency", action='store_true',
                     help="Read the starting frequency of template waveforms"
                          " from the template bank")
-parser.add_argument('--autogating-threshold', type=float, default=100)
+parser.add_argument('--autogating-threshold', type=float, default=None)
```

Comment:
Liting, you can remove the `default=None` here, since it is… the default :)

Reply:
Yeah sure! Done :)